LDM-151.tex

```diff
@@ -13,6 +13,27 @@
 
 \setcounter{secnumdepth}{5}
 
+%%%%%%%%%%%%%%%%%%%%%
+% Introduce mechanism to turn on and off various annotations (\XXX command,
+% \begin{notes}, and anything else bracketed by \ifannotated ... \fi)
+%
+\newif\ifannotated
+\annotatedtrue
+\annotatedfalse	% uncomment this to hide all annotations (comments, notes, etc)
+
+\ifannotated
+	% leave things as-is
+\else
+	% hide all \XXX commands
+	\renewcommand{\XXX}{}
+
+	% hide all \begin{note}...\end{note} text
+	\renewenvironment{note}[1][Note]
```

Comment:
I was wondering about this. Notes can sometimes be part of the documentation that people want to show so I was wondering about adding a version of note for developmental comments that would be disabled when `lsstdraft` mode is not in effect.